src/main/java/com/github/tommyettinger/digital/MathTools.java

```diff
@@ -53,6 +53,18 @@ private MathTools() {
      * false-positive equivalence with very small inputs.
      */
     public static final float FLOAT_ROUNDING_ERROR = 0x1p-20f; // was 0.000001f
+    
+    /**
+     * The smallest measurable difference between a given {@code float} value and a directly adjacent value.
```

Comment:
I don't think this is right; the smallest measurable difference varies a lot based on the magnitude of the floats, and would be measured as one ulp. This is, however, the smallest non-zero distance possible between two floats returned by `Random#nextFloat()`, if my math is right.